checklists/instructor-followup/instructor-trainer.md

```diff
@@ -0,0 +1,33 @@
+Immediately after students have completed instructor training please
+send this concluding email:
+
+    Thanks again for taking part in Software Carpentry instructor training
+    - we're pleased to have you on the team.  When you have a minute,
+    please:
+
+    1. Take a moment to go to AMY [1] and update your profile so we know where
+        you are and what you're comfortable teaching - that will help us
+        match you with workshops.
+
+    2. Send me a three-line bio and a photo to add to our website (see
+        our team page [2] for examples).
+
+    3. Make sure you're on the 'instructors' mailing list (signup at [3])
```

Comment:
I thought that we add new instructors to the 'instructors' mailing list right after they get their badge.